.github/ISSUE_TEMPLATE/Style.yml

```diff
@@ -0,0 +1,27 @@
+name: 👯‍♂️ Style Changing Request
+description: Suggest a style designs
+title: '[style]'
+labels: ["enhancement"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this template!
```

Comment:
I don't think that this block is necessary since this message is not appeared in the other 3 templates.